vocab/properties.json

```diff
@@ -791,14 +785,14 @@
         ]
     },
     "hasRequirement": {
-        "description": null,
+        "description": "Something essential to the existence or occurrence of something else.",
         "name": "Has requirement",
         "schemas": [
             "core/schemas/v3/products/softwareVersion.schema.tpl.json"
         ]
     },
     "hasResearchProducts": {
-        "description": null,
+        "description": "Reference to appendent research products.",
```

Comment:
"appendent" seems to be a legal term not normally used in everyday or scientific English. Would "subsidiary" work?

Reply:
Here I'll trust you fully. I'll change it to "subsidiary"